code/game/gamemodes/gang/gang.dm

```diff
@@ -69,18 +93,54 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 	if(gangs.len < 2) //Need at least two gangs
 		return 0
 
-	return 1
+	//turn off sec and captain
+	SSjob.DisableJob(/datum/job/captain)
+	SSjob.DisableJob(/datum/job/hos)
+	SSjob.DisableJob(/datum/job/warden)
+	SSjob.DisableJob(/datum/job/detective)
+	SSjob.DisableJob(/datum/job/officer)
 
+	return 1
 
-/datum/game_mode/gang/post_setup()
+/datum/game_mode/gang/send_intercept()
 	set waitfor = FALSE
-	..()
-	sleep(rand(10,100))
+	..(FALSE, FALSE)
+
+	priority_announce("Ballas Federation Announcement", "SECURITY SHOULD OF SIDED WITH THE BALLAS!", 'sound/machines/Alarm.ogg')
```

Comment:
Central Command Announcement: "WARNING! SECURITY HAS SUFFERED CATASTROPHIC DAMAGE., NO SURVIVORS. ENGINEERING PERSONNEL REPORT TO THE BRIG."

I would also suggest raising the station to Red Alert and declaring Martial Law.